dash/dash.py

```diff
@@ -492,7 +492,8 @@ def serve_component_suites(self, package_name, path_in_package_dist):
 
         mimetype = ({
             'js': 'application/JavaScript',
-            'css': 'text/css'
+            'css': 'text/css',
+            'map': 'application/json'
```

Comment:
What is the actual extension of those maps ? I remember seeing `.map.js`. In that case it would evaluate to `.js` since we take the last element of the split.

There's a stdlib package named `mimetypes` I would prefer if it would use that instead so we can support more than css/js but also images and other file types #96. It's missing a few types like javascript is guessed to `text/plain`, but we can add them with `mimetypes.add_type`.

Reply:
> What is the actual extension of those maps ? I remember seeing .map.js. In that case it would evaluate to .js since we take the last element of the split.

Correct source map extension is '.js.map'.

> There's a stdlib package named mimetypes I would prefer if it would use that instead so we can support more than css/js but also images and other file types #96. It's missing a few types like javascript is guessed to text/plain, but we can add them with mimetypes.add_type.

Using a library to manage mimetypes is a good idea in itself but is scope creep on what is a simple change.

Reply:
Agree that we should use mimetypes, but in a different PR.